packages/shared/lib/core/wallet/utils/outputs/isOutputExpired.ts

```diff
@@ -0,0 +1,21 @@
+import { CommonOutput, ExpirationUnlockCondition, UnlockConditionType } from '@iota/sdk/out/types'
+import { get } from 'svelte/store'
+import { getSlotIndexFromNodeInfo, nodeInfoProtocolParameters } from 'shared/lib/core/network'
+
+export function isOutputExpired(output: CommonOutput): boolean | undefined {
+    const expirationUnlockCondition = output.unlockConditions.find(
+        (unlockCondition) => unlockCondition.type === UnlockConditionType.Expiration
+    ) as ExpirationUnlockCondition
+    const nodeProtocolParameters = get(nodeInfoProtocolParameters)
+    const outputSlotIndex = expirationUnlockCondition?.slotIndex
+    if (!nodeProtocolParameters || !outputSlotIndex) return true
```

Comment:
Same as the other PR, why `true`? This would mark an output with no expiration unlock condition as if it was expired, it doesn't make sense unless I am missing something 👀 ❓